python/tests/conftest.py

```diff
@@ -1614,6 +1621,13 @@ def clear_query_stats():
     query_stats.reset_stats()
 
 
+@pytest.fixture(params=[1, 0])
+def recursive_normalizer_meta_structure_v2(request):
```

Comment:
nit: I think the name is confusing I initially thought that this enables only the V2 struct. I think `recursive_normalizer_meta_structure` is better.